www/src/templates/template-starter-page.js

```diff
@@ -28,10 +29,14 @@ class StarterTemplate extends React.Component {
       url: demoUrl,
       repo: repoUrl,
       fields: { starterShowcase },
-      childScreenshot = getFallbackScreenshot(fallback),
```

Comment:
childScreenshot wasn't undefined, it had a structure like:

```json
{
  "screenshotFile": null
}
```

so this fallback was never invoked. Reminder to myself to not get too clever next time :)